registered_span.go

```diff
@@ -1116,3 +1127,52 @@ func newAWSLambdaInvokeSpanData(span *spanS) AWSLambdaInvokeSpanData {
 
 	return data
 }
+
+// LogSpanData represents the `data` section of a logging span
+type LogSpanData struct {
+	SpanData
+	Tags LogSpanTags `json:"log"`
+}
+
+// NewLogSpanData initializes a new logging span data from tracer span
+func NewLogSpanData(span *spanS) LogSpanData {
```

Comment:
What do you think about renaming it to `newLogSpanData`, because we can not call `NewLogSpanData` from another package, due to its "private" argument `span *spanS`?

Reply:
Makes sense, will do 👍